cost-updater/src/main/scala/org/broadinstitute/dsp/workbench/hamm/costUpdater/CostUpdaterService.scala

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.dsp.workbench.hamm
+package costUpdater
+
+import cats.effect.Sync
+import cats.implicits._
+import fs2.concurrent.InspectableQueue
+import io.chrisdavenport.log4cats.Logger
+import io.circe.Encoder
+import org.http4s.HttpRoutes
+import org.http4s.circe.CirceEntityEncoder._
+import org.http4s.dsl.Http4sDsl
+import CostUpdaterService._
+
+class CostUpdaterService[F[_]: Sync: Logger, A](queue: InspectableQueue[F, A]) extends Http4sDsl[F] {
+
+  val service: HttpRoutes[F] = {
+    HttpRoutes.of[F] {
+      case GET -> Root / "status" =>
```

Comment:
I added `/version`. I'll leave the db status check out for now since this PR was mostly meant for hooking up the subscriber part in cost updater and replacing grpc. I updated `/status` for versions mostly just for having it handy for working on hamm-api-server deployment.